### ISSUE TITLE
Auto-inserting the Customer Account block makes the page look buggy in Amulet

### ISSUE THREAD

**Issue by dsas** [woothemes/woocommerce#48685]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The Amulet header looks off, it has a little silhouette of a man randomly floating around the header:
<img width="971" alt="Screenshot 2024-06-20 at 15 54 47" src="https://github.com/woocommerce/woocommerce/assets/93301/3476860a-b071-4d7e-958d-52bb1afa8732">

Upon investigation this is the `woocommerce/customer-account` block and it appears here due to a [block hook](https://github.com/woocommerce/woocommerce/blob/e5fafb455eded795d66ae0f75b2cc8a3a36320ca/plugins/woocommerce/src/Blocks/BlockTypes/CustomerAccount.php#L30C13-L30C36) inserting it after the navigation block. 

#46935 restricted this hook to only take effect on an allow list of themes. I can't immediately see how Amulet can be reasonably modified to make this work. Perhaps best to remove Amulet (and probably Tazza) from that allow list?

If customers ask about this, a workaround is possible by modifying [this code](https://github.com/woocommerce/woocommerce/pull/45581#issuecomment-2078017540) slightly.

This was initially reported in slack: p1718370754187519-slack-C048CUFRGFQ

CC @tjcafferkey, @danielwrobert

### Expected behavior

I didn't expect to see an icon at all - there's already a My account link in the navigation

### Actual behavior

As above.

### Steps to reproduce

1. Go to https://amuletdemo.wpcomstaging.com
2. Look at the header

(This happens when both amulet and woocommerce are activated)

### WordPress Environment

I'm using WordPress.com on the Creator plan, which has the latest released version of WooCommerce

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by tjcafferkey** [comment]
Thanks @dsas.

>If customers ask about this, a workaround is possible by modifying https://github.com/woocommerce/woocommerce/pull/45581#issuecomment-2078017540 slightly.

I wouldn't recommend they amend any code. They can make simply remove the block from the header template part and save it. Alternatively they could make an update I recommended below.

>I can't immediately see how Amulet can be reasonably modified to make this work.

We could wrap the navigation in a Row block which would have flex properties allowing the navigation and the my account block to align horizontally.

I'm going to add it to my list to open a PR for this theme making the above adjustment.

**Comment by dsas** [comment]
> simply remove the block from the header template part

Ha, ok that would indeed be loads simpler 😆, I didn't grok that it wouldn't be reinserted because the template is modified.

> We could wrap the navigation in a Row block which would have flex properties allowing the navigation and the my account block to align horizontally.

I think that might end up a bit odd still if the menu starts wrapping (many menu items / smaller viewports), worth a try though 👍 Let me know if you want a hand with anything.

**Comment by tjcafferkey** [comment]
>Ha, ok that would indeed be loads simpler 😆, I didn't grok that it wouldn't be reinserted because the template is modified.

Haha no problem! Yeah any block thats added via the block hooks API and then removed by the user wont be readded.

>I think that might end up a bit odd still if the menu starts wrapping

Indeed I can see this being an issue, but that said it's probably an issue with the menu in general as it will wrap with many menu items with or without the hooked block added.

![Screenshot 2024-06-21 at 09 04 38](https://github.com/woocommerce/woocommerce/assets/8639742/775c72fa-8ef0-4cd8-ae98-b9dc536431d1)

Also, unrelated to this issue specifically but strange that there is no wrapping container on the Amulet theme either?

![Screenshot 2024-06-21 at 09 06 03](https://github.com/woocommerce/woocommerce/assets/8639742/bc5aa73d-91ca-4a28-a11e-1b505dadad31)

**Comment by dsas** [comment]
> Also, unrelated to this issue specifically but strange that there is no wrapping container on the Amulet theme either?

Yep I found that odd too. There was some in the original design (p9Jlb4-64U-p2). It's covered by https://github.com/Automattic/themes/issues/7572

**Comment by gigitux** [comment]
@tjcafferkey do you think that we can close this issue given that it looks related to the theme?

**Comment by tjcafferkey** [comment]
@gigitux I think we should keep this open for the time being but mark it as low priority. It is a theme related bug, but caused by WooCommerce.
